pkg/agent/secondarynetwork/podwatch/controller.go

```diff
@@ -67,6 +67,12 @@ var (
 	getPodContainerDeviceIDs = cniserver.GetPodContainerDeviceIDs
 )
 
+// Structure to associate a unique VF's PCI Address to the Linux ethernet interface.
+type podContainerDeviceIDInfo struct {
```

Comment:
Should we just call it "podDeviceIDInfo" (I do not see why we used "podContainer" as names)?